src/data/text/item_descriptions.h

```diff
@@ -2025,57 +2050,57 @@ static const u8 sQuickPowderDesc[] = _(
     "powder boosts Speed.");
 
 static const u8 sFocusSashDesc[] = _(
-    "If the holder has\n"
-    "full HP, it endures\n"
-    "KO attacks with 1 HP.");
```

Comment:
i prefer the original's specificity about 1 hp

Reply:
Certainly, "prevents fainting" is a tad vague as it doesn't give you the exact aftermath of the effect.

For reference, I took inspiration from the description of the Focus Sash's cousin, the Focus Band.

I'm opened to suggestions 👀